specification/app/resource-manager/Microsoft.App/stable/2025-07-01/ManagedEnvironments.json

```diff
@@ -0,0 +1,1783 @@
+          "type": "string",
+          "readOnly": true,
+          "description": "The version of Keda"
+        }
+      }
+    },
+    "DaprConfiguration": {
+      "type": "object",
+      "description": "Configuration properties Dapr component",
+      "properties": {
+        "version": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The version of Dapr"
+        }
+      }
+    },
+    "VnetConfiguration": {
+      "type": "object",
+      "description": "Configuration properties for apps environment to join a Virtual Network",
+      "properties": {
+        "internal": {
+          "type": "boolean",
+          "description": "Boolean indicating the environment only has an internal load balancer. These environments do not have a public static IP resource. They must provide infrastructureSubnetId if enabling this property",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "infrastructureSubnetId": {
+          "type": "string",
+          "description": "Resource ID of a subnet for infrastructure components. Must not overlap with any other provided IP ranges.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "dockerBridgeCidr": {
+          "type": "string",
+          "description": "CIDR notation IP range assigned to the Docker bridge, network. Must not overlap with any other provided IP ranges.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "platformReservedCidr": {
+          "type": "string",
+          "description": "IP range in CIDR notation that can be reserved for environment infrastructure IP addresses. Must not overlap with any other provided IP ranges.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "platformReservedDnsIP": {
+          "type": "string",
+          "description": " An IP address from the IP range defined by platformReservedCidr that will be reserved for the internal DNS server.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        }
+      }
+    },
+    "Mtls": {
+      "type": "object",
+      "description": "Configuration properties for mutual TLS authentication",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Boolean indicating whether the mutual TLS authentication is enabled"
+        }
+      }
+    },
+    "IngressConfiguration": {
+      "description": "Settings for the ingress component, including workload profile, scaling, and connection handling.",
+      "type": "object",
+      "properties": {
+        "workloadProfileName": {
+          "description": "Name of the workload profile used by the ingress component. Required.",
+          "type": "string"
+        },
+        "terminationGracePeriodSeconds": {
```

Comment:
>terminationGracePeriodSeconds [](http://example.com/codeflow?start=9&length=29)

terminationGracePeriodInSeconds ?

Reply:
terminationGracePeriodSeconds already released in stable version [specification/app/resource-manager/Microsoft.App/stable/2022-03-01/CommonDefinitions.json](https://github.com/Azure/azure-rest-api-specs/blob/14849ffc5bc0c78ba942bcafa05e19e6162bb089/specification/app/resource-manager/Microsoft.App/stable/2022-03-01/CommonDefinitions.json#L220)

we cannot break change it